mmcls/models/losses/cross_entropy_loss.py

```diff
@@ -18,15 +18,63 @@ def cross_entropy(pred, label, weight=None, reduction='mean', avg_factor=None):
     return loss
 
 
+def binary_cross_entropy(pred,
+                         label,
+                         weight=None,
+                         reduction='mean',
+                         avg_factor=None):
+    """Calculate the binary CrossEntropy loss with logits.
+
+    Args:
+        pred (torch.Tensor): The prediction with shape (N, C), where C is the
+            number of classes.
+        label (torch.Tensor): The learning label with shape (N, C), where C is
+            the number of classes.
+        weight (torch.Tensor, optional): Weight of loss. Defaults to None.
+        reduction (str, optional): The method used to reduce the loss.
+            Options are "none", "mean" and "sum". Defaults to 'mean'.
+        avg_factor (int, optional): Average factor that is used to average
+            the loss. Defaults to None.
+
+    Returns:
+        torch.Tensor: The calculated loss
+    """
+    assert pred.dim() == label.dim()
+
+    loss = F.binary_cross_entropy_with_logits(pred, label, reduction='none')
```

Comment:
Both BCE  and  BCE_with_logits are for binary classification, but binary_cross_entropy_with_logits include sigmoid function and BCE. I think you should use `F.binary_cross_entropy`, or use an input arg to indicate use BCE or BCE_with_logits.